applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -1244,6 +1235,33 @@ void UPwSmallStrainElement<TDim,TNumNodes>::
     KRATOS_CATCH( "" )
 }
 
+//----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+template< unsigned int TDim, unsigned int TNumNodes >
+void UPwSmallStrainElement<TDim,TNumNodes>::
+    CalculatePermeabilityUpdateFactor( ElementVariables &rVariables)
+{
+    KRATOS_TRY
+    // KRATOS_INFO("0-SmallStrainUPwDiffOrderElement::CalculatePermeabilityUpdateFactor") << std::endl;
+
+    const PropertiesType& rProp = this->GetProperties();
+
+    if (rProp.Has(PERMEABILITY_CHANGE_INVERSE_FACTOR) &&
+        rProp[PERMEABILITY_CHANGE_INVERSE_FACTOR] > 0.0) {
```

Comment:
```suggestion
    if (rProp[PERMEABILITY_CHANGE_INVERSE_FACTOR] > 0.0) {
```

If `PERMEABILITY_CHANGE_INVERSE_FACTOR` is not defined, 0.0 will be returned

Reply:
I thought we receive a segmentation fault. Then, I can remove it